blog/2022-01-19-play-with-capact-via-cli.mdx

```diff
@@ -0,0 +1,334 @@
+
+To test it, execute [steps](#steps) once again, but now select the **Policy-Changed Execution** tab.
+
+## What is so special about it?
+
+Our example was simplified to a minimum, but you could see that you can:
+
+- chain different steps together,
+- and more importantly, write once and swap implementation details with ease.
+
+Capact is also great if you want to chain different tools together and keep the same entry point (UX) but that requires more complex examples. For now, we recommend you to take a look at our other examples described in the [**How can I get involved and learn more?**](#how-can-i-get-involved-and-learn-more) section.
+
+## Behind the wall
+
+As you saw, you can use CLI to browse Hub manifests, create a specific Action for a given Interface and retrieve the output. CLI communicates with our Gateway via GraphQL calls, same as the UI (Capact Dashboard). You can read more in the [E2E Architecture](/docs/next/architecture/e2e-architecture) document.
+
+However, a more important question is, how did we change the output message?
+
+<div className="line-numbers">
+
+```yaml {4-7,12} title="Implementation Manifest"
+spec:
+  # ... trimmed ...
+  imports:
+    - interfaceGroupPath: cap.interface.capactio.examples
+      alias: examples
+      methods:
+        - name: say
+
+  # ... trimmed ...
+  steps:
+    - - name: get-message
+        capact-action: examples.say # dynamic step, rendered by Engine based on the Policy.
+    - - name: print
+        template: print
+        arguments:
+          artifacts:
+            - name: message
+            from: "{{steps.get-message.outputs.artifacts.message}}"
+    # ... trimmed ...
+    - name: print
+      inputs:
+        artifacts:
+          - name: message
+          path: /tmp/message.yaml
+      container:
+        image: docker/whalesay:latest
+        command: [ sh, -c ]
+        args: [ "cowsay < /tmp/message.yaml" ]
+```
+
+</div>
+
+The snippet above shows the most important parts of the Implementation manifest. In lines **4-7**, we declare the import, which is later used in steps. In line **12**, we use **capact-action** to make a dynamic step with a given Interface.
+
+:::tip
+Instead of providing the full Interface path, we use **alias** assigned to the imported Interface.
+:::
+
+As a result, our Engine and, in particular, Argo Renderer has an option to put there an Implementation that fulfills the specified Interface. Thanks to that, this step is swappable based on the defined Policy.
+
+## How can I get involved and learn more?
+
+The [Getting Started](/docs/next/getting-started) goes into more detail about how to start with Capact and use its more-advanced features. There are also [Rocket.Chat](/docs/next/example/rocketchat-installation) and [Mattermost](/docs/next/example/mattermost-installation) tutorials that we keep up-to-date for you to walk you through using Capact in real cases.
```

Comment:
Please search the whole document and remove all `/next` occurrences as we should redirect to the latest stable version. I know that the CI will probably fail, but still - put a TODO somewhere in PR desc or something, to remember about that before merge 🙂 

```suggestion
The [Getting Started](/docs/getting-started) goes into more detail about how to start with Capact and use its more-advanced features. There are also [Rocket.Chat](/docs/example/rocketchat-installation) and [Mattermost](/docs/example/mattermost-installation) tutorials that we keep up-to-date for you to walk you through using Capact in real cases.
```